tests/testCases/positiveTC.spec.js

```diff
@@ -0,0 +1,26 @@
+const { test, expect } = require('@playwright/test');
+import { mainPage } from '../demoQA/pages/pageObject';
+import { filler } from '../demoQA/pages/inputs';
+
+test ('Positive scenario', async ({page}) => {
```

Comment:
Let's give more understandable names to tests, for example `Positive scenario: Fill form with correct data`